### PR TITLE
Feat/buttons dropdown

### PR DESCRIPTION
**Pull request by Andyderson**
Hey @rjgcabrera,

Mind if I ask for a PR from ya? It's mostly frontend features: dropdown menus, button, and styling.

Thanks!

### CODE REVIEW

File: client/src/dropdown.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+const options = [
+  'One', 'Two', 'Three', 'Four', 'Five', 'Six', 'Seven', 'Eight', 'Nine', 'Ten', 'Eleven', 'Twelve', 'Thirteen', 'Fourteen', 'Fifteen', 'More than fifteen'
+]
+
+class DropDownMenu extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      selected: options[0]
```
Comment (rjgcabrera) [inline client/src/dropdown.jsx]: Does this state denote the number of people in their party?

File: client/src/button.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+let Button = function (props) {
+    return (
+        <button className="button">Find a Table!</button>
```
Comment (rjgcabrera) [inline client/src/button.jsx]: click handler needed. it's okay to merge if this is just a basic skeleton, but send me another PR after implementation.

File: client/src/index.jsx [hunk 3]
```diff
@@ -22,8 +28,16 @@ class App extends React.Component {
...
-              <Calendar onChange={this.onChange} value={this.state.date}/>
+              <h1 className="header">Book a Reservation!</h1>
+              <h3>Party Size:</h3>
+                <DropDownMenu/>
+              <h3 className="time">Time:</h3>
+                <TimeDropDownMenu/>
+              <div><h3 className="date">Date:</h3>
+                <Calendar onChange={this.onChange} value={this.state.date}/>
+              </div>
+                <Button className="buttonSetting"/>
```
Comment (rjgcabrera) [inline client/src/index.jsx]: click handler property needed. it's okay to merge if this is just a basic skeleton, but send me another PR after implementation.

File: client/src/dropdown.jsx (hunk 1 above)
Comment (rjgcabrera) [reply client/src/dropdown.jsx]: Ah I see. Line 30 in this file answers my question. Add a comment here for clear documentation so integration is easy peasy.

File: client/src/index.jsx [hunk 4]
```diff
@@ -22,8 +28,16 @@ class App extends React.Component {
...
+              <h4>Booked 33 times today!</h4>
```
Comment (rjgcabrera) [inline client/src/index.jsx]: number of bookings should be dynamic based on your db. it's okay to merge if this is just a basic skeleton, but send me another PR after implementation.

Comment (Andyderson) [reply client/src/index.jsx]: You're correct! I should have added a comment signifying I'd change this to dynamic rendering later on. My bad.

File: client/src/index.jsx (hunk 3 above)
Comment (Andyderson) [reply client/src/index.jsx]: Will do, I haven't implemented functionality for the button yet. This button is going to be my popup modal.

File: client/src/button.jsx (hunk 2 above)
Comment (Andyderson) [reply client/src/button.jsx]: Same as below comment!

File: client/src/dropdown.jsx (hunk 1 above)
Comment (Andyderson) [reply client/src/dropdown.jsx]: Done!
